app/components/rate-goal/animations/cactus.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import GSAP from 'react-gsap-enhancer';
+import { TweenMax, TimelineMax, } from 'gsap';
+
+const createAnimation = ({target, options, }) => {
+
+  const bodyFrames = [
+    { bottom: '50px', height: '0px', },
+    { bottom: '54px', height: '50px', },
+    { bottom: '66px', height: '50px', },
+    { bottom: '70px', height: '60px', },
+    { bottom: '75px', height: '65px', },
+    { bottom: '75px', height: '70px', },
+    { bottom: '80px', height: '70px', },
+    { bottom: '85px', height: '70px', },
+    { bottom: '90px', height: '72px', },
+    { bottom: '95px', height: '77px', },
+    { bottom: '97px', height: '81px', },
+  ];
+
+  const armFrames = [
+    { visibility: 'hidden' },
+    { visibility: 'hidden' },
+    { visibility: 'hidden' },
+    { visibility: 'hidden' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+  ];
+
+  const vase = target.find({ id: 'vase' });
+  const cactus3 = target.find({ id: 'cactus3' });
+  const arms = target.find({ id: 'arms' });
+
+  let t1 = new TimelineMax();
+
+    const sequence = options.score > options.previousScore
+      ? bodyFrames.slice(options.previousScore, options.score + 1)
+      : bodyFrames.slice(options.score, options.previousScore + 1).reverse();
+
+      const armSequence = options.score > options.previousScore
```

Comment:
You have got a repeated function here,

getFrames:
```js
const getFrames: (allFrames, score, previousScore) => score > previousScore
  ? allFrames.slice(previousScore, score + 1)
  : allFrames.slice(score, previousScore + 1).reverse();
```

also, right now when score === previousScore, you go back one step.. Are you indexes okay?